src/components/wallet/transfer/TxList.vue

```diff
@@ -48,8 +50,10 @@ export default class TxList extends Vue {
     tx_list: ITransaction[] = []
     disabledAssets: AvaAsset[][] = []
     next_initial: AvaAsset | null = null
+    dummy = new BN(100)
```

Comment:
It seems that it has been introduced now but is not used anywhere - Does it serve any purpose?

Reply:
yes its not used, removed